docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -137,3 +137,14 @@ This allows you directly to SSH into the worker nodes, if required.
   # Worker nodes.
   $ ssh mycluster-worker1
   $ ssh mycluster-worker2
+
+
+Executing a Distributed Ray Program
```

Comment:
For discussion: We may want to add a dedicated guide on how to use Ray as part of the user job. That page can use a tip like this. A full YAML example https://github.com/skypilot-org/skypilot/issues/3195 may help promote this recommendation more directly.

Reply:
Yes, let’s add a yaml example of starting a distributed job with ray as mentioned above. We can try to move it to a separate page later after this pr is merged.

Reply:
So for confirmation: I should add the command to run the example in distributed_ray_train, the one that uses FashionMNIST and add the command for tha yaml file and its output as example in the documentation ?

Reply:
Yes, we can add the yaml file inline with the command to run that [yaml](https://github.com/skypilot-org/skypilot/blob/master/examples/distributed_ray_train/ray_train.yaml) : )